### PR TITLE
Add Amnezia VPN to Remote Access

### LINKED ISSUE #2930: Add AmneziaVPN
AmneziaVPN is one more project to easily deploy own VPN server using few different protocols. Currently they are:

- OpenVPN
- OpenVPN over Shadowsocks
- OpenVPN over Cloak* + Shadowsocks over Cloak*
- Wireguard
- IPSec/IKEv2

*- Cloak is obfuscation plugin (https://github.com/cbeuw/Cloak)

Like the Outline project, AmneziaVPN configures everything inside Docker containers to be able easily setup, reconfigure protocols and avoid polluting host system.

Project's profile on Github: https://github.com/amnezia-vpn
Desktop client: https://github.com/amnezia-vpn/desktop-client
Also already released Android client: https://play.google.com/store/apps/details?id=org.amnezia.vpn

### PR DESCRIPTION
**Pull request by kokomo123**
This should close #2930 as I believe Amnezia VPN might fall under this category. If I should move this to awesome-foss/awesome-sysadmin, I'd be happy to do it but as this issue is in this repository, I figure I would give it a shot. 

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [ ] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [ ] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [X] Additions are inserted preserving alphabetical order.
- [X] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [X] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [X] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [ ] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [X] Any software project you are adding to the list is actively maintained.
- [X] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1644,6 +1644,7 @@ Software and tools for managing [recipes](https://en.wikipedia.org/wiki/Recipe).
 
 [Remote desktop](https://en.wikipedia.org/wiki/Remote_desktop_software) and [SSH](https://en.wikipedia.org/wiki/Secure_Shell) software for remote management of computer systems.
 
+- [Amnezia VPN](https://github.com/amnezia-vpn/desktop-client) - Amnezia is a VPN client with the key feature of deploying your own VPN server on your virtual server. `GPL-3.0` `C++/C`
```
Comment (nodiscc) [inline README.md:1647]: The homepage is https://en.amnezia.org/

Comment (nodiscc) [reply README.md:1647]: https://github.com/amnezia-vpn/desktop-client is the source code for the desktop client, I could not find the server source code

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kokomo123) [reply README.md]: Fixed, thanks a bunch!

File: README.md [hunk 2]
```diff
@@ -1644,6 +1644,7 @@ Software and tools for managing [recipes](https://en.wikipedia.org/wiki/Recipe).
 
 [Remote desktop](https://en.wikipedia.org/wiki/Remote_desktop_software) and [SSH](https://en.wikipedia.org/wiki/Secure_Shell) software for remote management of computer systems.
 
+- [Amnezia VPN](https://en.amnezia.org) - Amnezia is a VPN client with the key feature of deploying your own VPN server on your virtual server. ([Source Code](https://github.com/amnezia-vpn/desktop-client)) `GPL-3.0` `C++/C`
```
Comment (nodiscc) [inline README.md:1647]: https://github.com/amnezia-vpn/desktop-client is the source code for the desktop client, but this link should be the source code for the server. I could not find it anywhere after a quick search.

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (kokomo123) [reply README.md:1647]: Could you explain that? so you need me to change the description or...

Comment (nodiscc) [reply README.md:1647]: You need to find the source code for the amnezia server (not the desktop client) and use that as a source code link.

Comment (kokomo123) [reply README.md:1647]: Noted.

Comment (nodiscc) [reply README.md:1647]: The problem is that users must be able to inspect the source code of what is being deployed to their machine. I couldn't find it after a quick search in the provided source code. So I don't think this should be added in the current state.

Comment (kokomo123) [reply README.md:1647]: Alright, I shall close this one as well. Apologies for adding this one.

Comment (pokamest) [reply README.md:1647]: You can find all scripts here - https://github.com/amnezia-vpn/desktop-client/tree/master/client/server_scripts
Also you can check pentest report - https://7asecurity.com/reports/pentest-report-amneziavpn.pdf

**Pull request closed without merging**